.env_sample

```diff
@@ -0,0 +1 @@
+SECRET_KEY=<your_secret_key>
```

Comment:
Seems like you don't need this file

Reply:
This file is for easier deployment of the project on a local machine.  It is mentioned in the readme in item 4 in the Setup section. To run the project, this file needs to be renamed to .env and the generated key must be placed in it.